### ISSUE TITLE
Ignoring windows for mouse_follows_focus via rules

### ISSUE THREAD

**Issue by dominiklohmann** [koekeishiya/yabai#675]
I'd like to be able to ignore windows for `mouse_follows_focus` via rules.

Rationale: iStat Menus is unusable on Big Sur, because its menubar dropdown windows now trigger `mouse_follows_focus`.

Ideally, a rule like `yabai -m rule --add='^iStat Menus Status$' mouse_follows_focus=off` would allow for selectively enabling or disabling `mouse_follows_focus`, overriding the default set by `yabai -m config mouse_follows_focus on|off`.

**Comment by koekeishiya** [comment]
Fixed on master.

**Comment by peppy** [comment]
I uninstalled iStat Menus due to a memory leak, but this worked for me when I was using it:

`yabai -m rule --add app="iStat Menus.*" manage=off mouse_follows_focus=off`

**Comment by majordoobie** [comment]
Thanks for the reply! But no dice. I was not aware of any leaks... What are you using now?

**Comment by peppy** [comment]
If you're seeing issues specifically with the `mouse_follows_focus` part, make sure to rebuild from latest `master` as an issue with this flag was fixed very recently (https://github.com/koekeishiya/yabai/issues/1267).

The memory leak for me was with very specific to AMD GPUs, and saw a constant vRAM leak over hours until it would hard crash my system (narrowed down to when iStat Menus was visible in the menu bar). Probably not relevant to most users. I have no need for a replacement, although I did come across [this app](https://www.seense.com/menubarstats/) recently.
